packages/okta-react/src/Auth.js

```diff
@@ -22,7 +22,16 @@ const containsAuthTokens = /id_token|access_token|code/;
 
 export default class Auth {
   constructor(config) {
-    assertIssuer(config.issuer);
+    const testConfig = Object.create(config);
```

Comment:
We probably do not need to copy over the contents within `config`, since we're only using it in one place.

This can be simplified to something like:

```javascript
const testing = {
  // If the config is undefined, cast it to false
  disableHttpsCheck: !!config.disableHttpsCheck
};

assertIssuer(config.issuer, testing);
```